README.md

```diff
@@ -3190,7 +3190,7 @@ require'lspconfig'.omnisharp.setup{}
   Commands:
   
   Default Values:
-    cmd = { "/home/runner/.cache/nvim/lspconfig/omnisharp/run", "--languageserver", "--hostPID", "2524" }
+    cmd = { "/home/runner/.cache/nvim/lspconfig/omnisharp/run", "--languageserver", "--hostPID", "2517" }
```

Comment:
this appears in all your PRs xD

Reply:
GHA docgen hates port 2524 for some reason🤷‍♀️

Reply:
ha I see the command is non-stable `cmd = installer.info().cmd;` xD